packages/react/src/vertex3.tsx

```diff
@@ -91,7 +96,7 @@ export const Vertex3: React.FunctionComponent<IVertex3> = ({
         fullAnnotationWidth += annotationGutter * (annotations.length - 1);
     }
     const textElement = <g transform={`translate(${textOffsetX} ${annoOffsetY})`}>
-        { text === "" ? null : <TextBox
+        {!showLabel || text === "" ? <circle r={noLabelRadius} stroke={textboxStroke} fill={textFill} /> : <TextBox
```

Comment:
Show a circle where the label would have been?

Reply:
Yup - more a dot.